### ISSUE TITLE
Proper installation process

### ISSUE THREAD

**Issue by maxdd** [mancuniancol/plugin.video.horriblesubsviewer#1]
Hello, fortunately you will bring this amazing addons to us.

I currently installed

https://github.com/mancuniancol/script.module.mctools

and the current addon.

The problem is that the Viewer is only showing me Add releases or Add a new Anime .
I tried to add something but i get the following error:

17:25:30 T:3116  NOTICE: [xbmcswift2] Request for "/releases" matches rule for function "releases"
17:25:35 T:5152   ERROR: XFILE::CDirectory::GetDirectory - Error getting plugin://plugin.video.horriblesubsviewer/releases
17:25:35 T:5152   ERROR: CGUIMediaWindow::GetDirectory(plugin://plugin.video.horriblesubsviewer/releases) failed
17:25:36 T:10572  NOTICE: [xbmcswift2] Request for "/" matches rule for function "index"

and 

17:26:30 T:9884  NOTICE: [HorribleSubs Viewer] http://horriblesubs.info/lib/search.php?value=Bloodivores&nextid=0
17:26:35 T:5152   ERROR: XFILE::CDirectory::GetDirectory - Error getting plugin://plugin.video.horriblesubsviewer/add
17:26:35 T:5152   ERROR: CGUIMediaWindow::GetDirectory(plugin://plugin.video.horriblesubsviewer/add) failed
17:26:36 T:8508  NOTICE: [xbmcswift2] Request for "/" matches rule for function "index"

**Comment by maxdd** [comment]
I've tried through this repo as well
repository.torrentstreaming

But now it just hangs

**Comment by mancuniancol** [comment]
I am not close to my pc. But, I will check it. Horriblesubs page changes often. Then, I need to keep update it

**Comment by mancuniancol** [comment]
Actually, this is the official repo https://github.com/mancuniancol/repository.Torrent-Viewers-Tools/archive/v1.4.1.zip
I've done some updates last weekend

**Comment by maxdd** [comment]
Horriblesubs version is 1.2.1 with your official repo but same error, same goes for haruichan

**Comment by mancuniancol** [comment]
Good to know, I will check it. Haurichan API doesn't work anymore.

**Comment by maxdd** [comment]
Why is that? Will it ever come back?

**Comment by mancuniancol** [comment]
Not sure. But, the rss feed I think still works with the rss viewer

**Comment by maxdd** [comment]
Below a debug log

09:42:03 T:8156   DEBUG: CGUIMediaWindow::GetDirectory (plugin://plugin.video.horriblesubsviewer/releases)
09:42:03 T:8156   DEBUG:   ParentPath = [plugin://plugin.video.horriblesubsviewer/]
09:42:03 T:3488   DEBUG: Thread JobWorker start, auto delete: true
09:42:03 T:3488   DEBUG: XFILE::CPluginDirectory::StartScript - calling plugin [COLOR FF533772]HorribleSubs[/COLOR] Viewer('plugin://plugin.video.horriblesubsviewer/releases','9','')
09:42:03 T:3488   DEBUG: XFILE::CPluginDirectory::WaitOnScriptResult - waiting on the [COLOR FF533772]HorribleSubs[/COLOR] Viewer (id=23) plugin...
09:42:03 T:4412   DEBUG: Thread LanguageInvoker start, auto delete: false
09:42:03 T:4412    INFO: initializing python engine.
09:42:03 T:4412   DEBUG: CPythonInvoker(23, C:\Users\maxdd\AppData\Roaming\Kodi\addons\plugin.video.horriblesubsviewer\main.py): start processing
09:42:03 T:4412   DEBUG: -->Python Interpreter Initialized<--
09:42:03 T:4412   DEBUG: CPythonInvoker(23, C:\Users\maxdd\AppData\Roaming\Kodi\addons\plugin.video.horriblesubsviewer\main.py): the source file to load is "C:\Users\maxdd\AppData\Roaming\Kodi\addons\plugin.video.horriblesubsviewer\main.py"
09:42:03 T:4412   DEBUG: CPythonInvoker(23, C:\Users\maxdd\AppData\Roaming\Kodi\addons\plugin.video.horriblesubsviewer\main.py): setting the Python path to C:\Users\maxdd\AppData\Roaming\Kodi\addons\plugin.video.horriblesubsviewer;C:\Users\maxdd\AppData\Roaming\Kodi\addons\script.module.addon.common\lib;C:\Users\maxdd\AppData\Roaming\Kodi\addons\script.module.beautifulsoup4\lib;C:\Users\maxdd\AppData\Roaming\Kodi\addons\script.module.feedparser\lib;C:\Users\maxdd\AppData\Roaming\Kodi\addons\script.module.html5lib\lib;C:\Users\maxdd\AppData\Roaming\Kodi\addons\script.module.httplib2\lib;C:\Users\maxdd\AppData\Roaming\Kodi\addons\script.module.mctools-master\lib;C:\Users\maxdd\AppData\Roaming\Kodi\addons\script.module.metahandler\lib;C:\Users\maxdd\AppData\Roaming\Kodi\addons\script.module.myconnpy\lib;C:\Users\maxdd\AppData\Roaming\Kodi\addons\script.module.requests\lib;C:\Users\maxdd\AppData\Roaming\Kodi\addons\script.module.simplejson\lib;C:\Users\maxdd\AppData\Roaming\Kodi\addons\script.module.six\lib;C:\Users\maxdd\AppData\Roaming\Kodi\addons\script.module.xbmcswift2\lib;C:\Program Files (x86)\Kodi\system\python\DLLs;C:\Program Files (x86)\Kodi\system\python\Lib;C:\Program Files (x86)\Kodi\python27.zip;C:\Program Files (x86)\Kodi\system\python\lib\plat-win;C:\Program Files (x86)\Kodi\system\python\lib\lib-tk;C:\Program Files (x86)\Kodi;C:\Program Files (x86)\Kodi\system\python;C:\Program Files (x86)\Kodi\system\python\lib\site-packages
09:42:03 T:4412   DEBUG: CPythonInvoker(23, C:\Users\maxdd\AppData\Roaming\Kodi\addons\plugin.video.horriblesubsviewer\main.py): entering source directory C:\Users\maxdd\AppData\Roaming\Kodi\addons\plugin.video.horriblesubsviewer
09:42:03 T:4412   DEBUG: CPythonInvoker(23, C:\Users\maxdd\AppData\Roaming\Kodi\addons\plugin.video.horriblesubsviewer\main.py): instantiating addon using automatically obtained id of "plugin.video.horriblesubsviewer" dependent on version 2.12.0 of the xbmc.python api
09:42:03 T:8156   DEBUG: ------ Window Init (DialogBusy.xml) ------
09:42:03 T:8156   DEBUG: CInputManager::ProcessMouse: trying mouse action doubleclick
09:42:04 T:4412   DEBUG: [xbmcswift2] Adding url rule "/" named "index" pointing to function "index"
09:42:04 T:4412   DEBUG: [xbmcswift2] Adding url rule "/add" named "add" pointing to function "add"
09:42:04 T:4412   DEBUG: [xbmcswift2] Adding url rule "/remove/<name>" named "remove" pointing to function "remove"
09:42:04 T:4412   DEBUG: [xbmcswift2] Adding url rule "/releases" named "releases" pointing to function "releases"
09:42:04 T:4412   DEBUG: [xbmcswift2] Adding url rule "/readUrl/<url>/" named "readUrl" pointing to function "readUrl"
09:42:04 T:4412   DEBUG: [xbmcswift2] Adding url rule "/play/<magnet>" named "play" pointing to function "play"
09:42:04 T:4412   DEBUG: [xbmcswift2] Adding url rule "/importAll/<name>" named "importAll" pointing to function "importAll"
09:42:04 T:4412   DEBUG: [xbmcswift2] Adding url rule "/importOne/<title>/<magnet>" named "importOne" pointing to function "importOne"
09:42:04 T:4412   DEBUG: [xbmcswift2] Adding url rule "/unsubscribe/<name>" named "unsubscribe" pointing to function "unsubscribe"
09:42:04 T:4412   DEBUG: [xbmcswift2] Adding url rule "/rebuilt/<name>" named "rebuilt" pointing to function "rebuilt"
09:42:04 T:4412   DEBUG: [xbmcswift2] Handling incoming request for /releases
09:42:04 T:4412  NOTICE: [xbmcswift2] Request for "/releases" matches rule for function "releases"
09:42:04 T:4412   DEBUG: POParser: PO file has Win Style Line Endings. Converted in memory to Linux LF for file: C:\Users\maxdd\AppData\Roaming\Kodi\addons\plugin.video.horriblesubsviewer\resources\language\English\strings.po
09:42:04 T:4412   DEBUG: POParser: loaded 105 strings from file C:\Users\maxdd\AppData\Roaming\Kodi\addons\plugin.video.horriblesubsviewer\resources\language\English\strings.po
09:42:04 T:8156   DEBUG: ------ Window Init (DialogKaiToast.xml) ------
09:42:04 T:5792    INFO: CActiveAESink::OpenSink - initialize sink
09:42:04 T:5792   DEBUG: CActiveAESink::OpenSink - trying to open device DIRECTSOUND:default
09:42:04 T:8156   DEBUG: SECTION:LoadDLL(special://xbmcbin/system/ImageLib.dll)
09:42:04 T:5792   DEBUG: CAESinkDirectSound::Initialize: Using Window handle: 0004037E
09:42:04 T:5792   DEBUG: CAESinkDirectSound::Initialize: secondary buffer created
09:42:04 T:5792   DEBUG: CAESinkDirectSound::Initialize: Initializing DirectSound with the following parameters:
09:42:04 T:5792   DEBUG:   Audio Device    : Altoparlanti (Sound Blaster Extigy)
09:42:04 T:5792   DEBUG:   Sample Rate     : 44100
09:42:04 T:5792   DEBUG:   Sample Format   : AE_FMT_FLOAT
09:42:04 T:5792   DEBUG:   Bits Per Sample : 32
09:42:04 T:5792   DEBUG:   Valid Bits/Samp : 32
09:42:04 T:5792   DEBUG:   Channel Count   : 2
09:42:04 T:5792   DEBUG:   Block Align     : 8
09:42:04 T:5792   DEBUG:   Avg. Bytes Sec  : 352800
09:42:04 T:5792   DEBUG:   Samples/Block   : 32
09:42:04 T:5792   DEBUG:   Format cBSize   : 22
09:42:04 T:5792   DEBUG:   Channel Layout  : FL,FR
09:42:04 T:5792   DEBUG:   Channel Mask    : 3
09:42:04 T:5792   DEBUG:   Frames          : 661
09:42:04 T:5792   DEBUG:   Frame Samples   : 1322
09:42:04 T:5792   DEBUG:   Frame Size      : 8
09:42:04 T:5792   DEBUG: CActiveAESink::OpenSink - DIRECTSOUND Initialized:
09:42:04 T:5792   DEBUG:   Output Device : default
09:42:04 T:5792   DEBUG:   Sample Rate   : 44100
09:42:04 T:5792   DEBUG:   Sample Format : AE_FMT_FLOAT
09:42:04 T:5792   DEBUG:   Channel Count : 2
09:42:04 T:5792   DEBUG:   Channel Layout: FL,FR
09:42:04 T:5792   DEBUG:   Frames        : 661
09:42:04 T:5792   DEBUG:   Frame Samples : 1322
09:42:04 T:5792   DEBUG:   Frame Size    : 8
09:42:04 T:5792   DEBUG: CAESinkDirectSound::CheckPlayStatus: Resuming Playback
09:42:04 T:4412    INFO: CPythonInvoker(23, C:\Users\maxdd\AppData\Roaming\Kodi\addons\plugin.video.horriblesubsviewer\main.py): script successfully run
09:42:04 T:3488   DEBUG: XFILE::CPluginDirectory::WaitOnScriptResult- plugin returned failure
09:42:04 T:8156   ERROR: XFILE::CDirectory::GetDirectory - Error getting plugin://plugin.video.horriblesubsviewer/releases
09:42:04 T:8156   ERROR: CGUIMediaWindow::GetDirectory(plugin://plugin.video.horriblesubsviewer/releases) failed
09:42:04 T:8156   DEBUG: CGUIMediaWindow::GetDirectory (plugin://plugin.video.horriblesubsviewer/)
09:42:04 T:8156   DEBUG:   ParentPath = [addons://sources/video]
09:42:04 T:8156   DEBUG: Loading items: 2, directory: plugin://plugin.video.horriblesubsviewer/ sort method: 0, ascending: false
09:42:04 T:4444   DEBUG: Thread BackgroundLoader start, auto delete: false
09:42:04 T:4444   DEBUG: Thread BackgroundLoader 4444 terminating
09:42:04 T:4412    INFO: Python script stopped
09:42:04 T:4412   DEBUG: Thread LanguageInvoker 4412 terminating
09:42:04 T:8156   DEBUG: ------ Window Deinit (DialogBusy.xml) ------
09:42:04 T:8156   DEBUG: CInputManager::ProcessMouse: trying mouse action leftclick
09:42:04 T:8156   DEBUG: CGUIMediaWindow::GetDirectory (plugin://plugin.video.horriblesubsviewer/add)
09:42:04 T:8156   DEBUG:   ParentPath = [plugin://plugin.video.horriblesubsviewer/]
09:42:04 T:3488   DEBUG: XFILE::CPluginDirectory::StartScript - calling plugin [COLOR FF533772]HorribleSubs[/COLOR] Viewer('plugin://plugin.video.horriblesubsviewer/add','10','')
09:42:04 T:3488   DEBUG: XFILE::CPluginDirectory::WaitOnScriptResult - waiting on the [COLOR FF533772]HorribleSubs[/COLOR] Viewer (id=24) plugin...
09:42:04 T:4492   DEBUG: Thread LanguageInvoker start, auto delete: false
09:42:04 T:4492    INFO: initializing python engine.
09:42:04 T:4492   DEBUG: CPythonInvoker(24, C:\Users\maxdd\AppData\Roaming\Kodi\addons\plugin.video.horriblesubsviewer\main.py): start processing
09:42:04 T:4492   DEBUG: -->Python Interpreter Initialized<--
09:42:04 T:4492   DEBUG: CPythonInvoker(24, C:\Users\maxdd\AppData\Roaming\Kodi\addons\plugin.video.horriblesubsviewer\main.py): the source file to load is "C:\Users\maxdd\AppData\Roaming\Kodi\addons\plugin.video.horriblesubsviewer\main.py"
09:42:04 T:4492   DEBUG: CPythonInvoker(24, C:\Users\maxdd\AppData\Roaming\Kodi\addons\plugin.video.horriblesubsviewer\main.py): setting the Python path to C:\Users\maxdd\AppData\Roaming\Kodi\addons\plugin.video.horriblesubsviewer;C:\Users\maxdd\AppData\Roaming\Kodi\addons\script.module.addon.common\lib;C:\Users\maxdd\AppData\Roaming\Kodi\addons\script.module.beautifulsoup4\lib;C:\Users\maxdd\AppData\Roaming\Kodi\addons\script.module.feedparser\lib;C:\Users\maxdd\AppData\Roaming\Kodi\addons\script.module.html5lib\lib;C:\Users\maxdd\AppData\Roaming\Kodi\addons\script.module.httplib2\lib;C:\Users\maxdd\AppData\Roaming\Kodi\addons\script.module.mctools-master\lib;C:\Users\maxdd\AppData\Roaming\Kodi\addons\script.module.metahandler\lib;C:\Users\maxdd\AppData\Roaming\Kodi\addons\script.module.myconnpy\lib;C:\Users\maxdd\AppData\Roaming\Kodi\addons\script.module.requests\lib;C:\Users\maxdd\AppData\Roaming\Kodi\addons\script.module.simplejson\lib;C:\Users\maxdd\AppData\Roaming\Kodi\addons\script.module.six\lib;C:\Users\maxdd\AppData\Roaming\Kodi\addons\script.module.xbmcswift2\lib;C:\Program Files (x86)\Kodi\system\python\DLLs;C:\Program Files (x86)\Kodi\system\python\Lib;C:\Program Files (x86)\Kodi\python27.zip;C:\Program Files (x86)\Kodi\system\python\lib\plat-win;C:\Program Files (x86)\Kodi\system\python\lib\lib-tk;C:\Program Files (x86)\Kodi;C:\Program Files (x86)\Kodi\system\python;C:\Program Files (x86)\Kodi\system\python\lib\site-packages
09:42:04 T:4492   DEBUG: CPythonInvoker(24, C:\Users\maxdd\AppData\Roaming\Kodi\addons\plugin.video.horriblesubsviewer\main.py): entering source directory C:\Users\maxdd\AppData\Roaming\Kodi\addons\plugin.video.horriblesubsviewer
09:42:04 T:4492   DEBUG: CPythonInvoker(24, C:\Users\maxdd\AppData\Roaming\Kodi\addons\plugin.video.horriblesubsviewer\main.py): instantiating addon using automatically obtained id of "plugin.video.horriblesubsviewer" dependent on version 2.12.0 of the xbmc.python api
09:42:05 T:8156   DEBUG: ------ Window Init (DialogBusy.xml) ------
09:42:05 T:8156   DEBUG: CInputManager::ProcessMouse: trying mouse action doubleclick
09:42:05 T:4492   DEBUG: [xbmcswift2] Adding url rule "/" named "index" pointing to function "index"
09:42:05 T:4492   DEBUG: [xbmcswift2] Adding url rule "/add" named "add" pointing to function "add"
09:42:05 T:4492   DEBUG: [xbmcswift2] Adding url rule "/remove/<name>" named "remove" pointing to function "remove"
09:42:05 T:4492   DEBUG: [xbmcswift2] Adding url rule "/releases" named "releases" pointing to function "releases"
09:42:05 T:4492   DEBUG: [xbmcswift2] Adding url rule "/readUrl/<url>/" named "readUrl" pointing to function "readUrl"
09:42:05 T:4492   DEBUG: [xbmcswift2] Adding url rule "/play/<magnet>" named "play" pointing to function "play"
09:42:05 T:4492   DEBUG: [xbmcswift2] Adding url rule "/importAll/<name>" named "importAll" pointing to function "importAll"
09:42:05 T:4492   DEBUG: [xbmcswift2] Adding url rule "/importOne/<title>/<magnet>" named "importOne" pointing to function "importOne"
09:42:05 T:4492   DEBUG: [xbmcswift2] Adding url rule "/unsubscribe/<name>" named "unsubscribe" pointing to function "unsubscribe"
09:42:05 T:4492   DEBUG: [xbmcswift2] Adding url rule "/rebuilt/<name>" named "rebuilt" pointing to function "rebuilt"
09:42:05 T:4492   DEBUG: [xbmcswift2] Handling incoming request for /add
09:42:05 T:4492  NOTICE: [xbmcswift2] Request for "/add" matches rule for function "add"
09:42:05 T:8156   DEBUG: ------ Window Init (DialogKeyboard.xml) ------
09:42:05 T:8156   DEBUG: CAnnouncementManager - Announcement: OnInputRequested from xbmc
09:42:05 T:8156   DEBUG: GOT ANNOUNCEMENT, type: 128, from xbmc, message OnInputRequested
09:42:05 T:8156   DEBUG: ------ Window Deinit (DialogKaiToast.xml) ------
09:42:06 T:8156   DEBUG: Keyboard: scancode: 0x2a, sym: 0x0130, unicode: 0x0000, modifier: 0x1
09:42:06 T:8156   DEBUG: CInputManager::OnKey: leftshift (0xf0d2) pressed, trying keyboard action f0d2
09:42:06 T:8156   DEBUG: CInputManager::OnKey: leftshift (0xf0d2) pressed, action is
09:42:06 T:8156   DEBUG: ------ Window Deinit (Pointer.xml) ------
09:42:07 T:8156   DEBUG: Keyboard: scancode: 0x2a, sym: 0x0130, unicode: 0x0000, modifier: 0x1
09:42:07 T:8156   DEBUG: CInputManager::OnKey: shift-long-leftshift (0x102f0d2) pressed, trying keyboard action f0d2
09:42:07 T:8156   DEBUG: CInputManager::OnKey: shift-long-leftshift (0x102f0d2) pressed, action is
09:42:07 T:8156   DEBUG: Keyboard: scancode: 0x2a, sym: 0x0130, unicode: 0x0000, modifier: 0x1
09:42:07 T:8156   DEBUG: CInputManager::OnKey: shift-long-leftshift (0x102f0d2) pressed, trying keyboard action f0d2
09:42:07 T:8156   DEBUG: CInputManager::OnKey: shift-long-leftshift (0x102f0d2) pressed, action is
09:42:07 T:8156   DEBUG: Keyboard: scancode: 0x2a, sym: 0x0130, unicode: 0x0000, modifier: 0x1
09:42:07 T:8156   DEBUG: CInputManager::OnKey: shift-long-leftshift (0x102f0d2) pressed, trying keyboard action f0d2
09:42:07 T:8156   DEBUG: CInputManager::OnKey: shift-long-leftshift (0x102f0d2) pressed, action is
09:42:07 T:8156   DEBUG: Keyboard: scancode: 0x2a, sym: 0x0130, unicode: 0x0000, modifier: 0x1
09:42:07 T:8156   DEBUG: CInputManager::OnKey: shift-long-leftshift (0x102f0d2) pressed, trying keyboard action f0d2
09:42:07 T:8156   DEBUG: CInputManager::OnKey: shift-long-leftshift (0x102f0d2) pressed, action is
09:42:07 T:8156   DEBUG: Keyboard: scancode: 0x2a, sym: 0x0130, unicode: 0x0000, modifier: 0x1
09:42:07 T:8156   DEBUG: CInputManager::OnKey: shift-long-leftshift (0x102f0d2) pressed, trying keyboard action f0d2
09:42:07 T:8156   DEBUG: CInputManager::OnKey: shift-long-leftshift (0x102f0d2) pressed, action is
09:42:07 T:8156   DEBUG: Keyboard: scancode: 0x2a, sym: 0x0130, unicode: 0x0000, modifier: 0x1
09:42:07 T:8156   DEBUG: CInputManager::OnKey: shift-long-leftshift (0x102f0d2) pressed, trying keyboard action f0d2
09:42:07 T:8156   DEBUG: CInputManager::OnKey: shift-long-leftshift (0x102f0d2) pressed, action is
09:42:07 T:8156   DEBUG: Keyboard: scancode: 0x2a, sym: 0x0130, unicode: 0x0000, modifier: 0x1
09:42:07 T:8156   DEBUG: CInputManager::OnKey: shift-long-leftshift (0x102f0d2) pressed, trying keyboard action f0d2
09:42:07 T:8156   DEBUG: CInputManager::OnKey: shift-long-leftshift (0x102f0d2) pressed, action is
09:42:07 T:8156   DEBUG: Keyboard: scancode: 0x2a, sym: 0x0130, unicode: 0x0000, modifier: 0x1
09:42:07 T:8156   DEBUG: CInputManager::OnKey: shift-long-leftshift (0x102f0d2) pressed, trying keyboard action f0d2
09:42:07 T:8156   DEBUG: CInputManager::OnKey: shift-long-leftshift (0x102f0d2) pressed, action is
09:42:07 T:8156   DEBUG: Keyboard: scancode: 0x2a, sym: 0x0130, unicode: 0x0000, modifier: 0x1
09:42:07 T:8156   DEBUG: CInputManager::OnKey: shift-long-leftshift (0x102f0d2) pressed, trying keyboard action f0d2
09:42:07 T:8156   DEBUG: CInputManager::OnKey: shift-long-leftshift (0x102f0d2) pressed, action is
09:42:07 T:8156   DEBUG: Keyboard: scancode: 0x2a, sym: 0x0130, unicode: 0x0000, modifier: 0x1
09:42:07 T:8156   DEBUG: CInputManager::OnKey: shift-long-leftshift (0x102f0d2) pressed, trying keyboard action f0d2
09:42:07 T:8156   DEBUG: CInputManager::OnKey: shift-long-leftshift (0x102f0d2) pressed, action is
09:42:07 T:8156   DEBUG: Keyboard: scancode: 0x23, sym: 0x0068, unicode: 0x0048, modifier: 0x1
09:42:07 T:8156   DEBUG: CInputManager::OnKey: shift-h (0x2f048) pressed, trying keyboard action f148
09:42:07 T:8156   DEBUG: Keyboard: scancode: 0x1e, sym: 0x0061, unicode: 0x0061, modifier: 0x0
09:42:07 T:8156   DEBUG: CInputManager::OnKey: a (0xf041) pressed, trying keyboard action f161
09:42:07 T:8156   DEBUG: Keyboard: scancode: 0x17, sym: 0x0069, unicode: 0x0069, modifier: 0x0
09:42:07 T:8156   DEBUG: CInputManager::OnKey: i (0xf049) pressed, trying keyboard action f169
09:42:07 T:8156   DEBUG: Keyboard: scancode: 0x25, sym: 0x006b, unicode: 0x006b, modifier: 0x0
09:42:07 T:8156   DEBUG: CInputManager::OnKey: k (0xf04b) pressed, trying keyboard action f16b
09:42:08 T:8156   DEBUG: Keyboard: scancode: 0x15, sym: 0x0079, unicode: 0x0079, modifier: 0x0
09:42:08 T:8156   DEBUG: CInputManager::OnKey: y (0xf059) pressed, trying keyboard action f179
09:42:08 T:8156   DEBUG: Keyboard: scancode: 0x16, sym: 0x0075, unicode: 0x0075, modifier: 0x0
09:42:08 T:8156   DEBUG: CInputManager::OnKey: u (0xf055) pressed, trying keyboard action f175
09:42:08 T:8156   DEBUG: Keyboard: scancode: 0x16, sym: 0x0075, unicode: 0x0075, modifier: 0x0
09:42:08 T:8156   DEBUG: CInputManager::OnKey: u (0xf055) pressed, trying keyboard action f175
09:42:09 T:8156   DEBUG: Keyboard: scancode: 0x1c, sym: 0x000d, unicode: 0x000d, modifier: 0x0
09:42:09 T:8156   DEBUG: CInputManager::OnKey: return (0xf00d) pressed, trying keyboard action 7
09:42:09 T:8156   DEBUG: ------ Window Deinit (DialogKeyboard.xml) ------
09:42:09 T:8156   DEBUG: CAnnouncementManager - Announcement: OnInputFinished from xbmc
09:42:09 T:8156   DEBUG: GOT ANNOUNCEMENT, type: 128, from xbmc, message OnInputFinished
09:42:09 T:4492   DEBUG: POParser: PO file has Win Style Line Endings. Converted in memory to Linux LF for file: C:\Users\maxdd\AppData\Roaming\Kodi\addons\plugin.video.horriblesubsviewer\resources\language\English\strings.po
09:42:09 T:4492   DEBUG: POParser: loaded 105 strings from file C:\Users\maxdd\AppData\Roaming\Kodi\addons\plugin.video.horriblesubsviewer\resources\language\English\strings.po
09:42:09 T:4492  NOTICE: [HorribleSubs Viewer] http://horriblesubs.info/lib/search.php?value=Haikyuu&nextid=0
09:42:09 T:8156   DEBUG: ------ Window Init (DialogKaiToast.xml) ------
09:42:09 T:4492    INFO: CPythonInvoker(24, C:\Users\maxdd\AppData\Roaming\Kodi\addons\plugin.video.horriblesubsviewer\main.py): script successfully run
09:42:09 T:3488   DEBUG: XFILE::CPluginDirectory::WaitOnScriptResult- plugin returned failure
09:42:09 T:8156   ERROR: XFILE::CDirectory::GetDirectory - Error getting plugin://plugin.video.horriblesubsviewer/add
09:42:09 T:8156   ERROR: CGUIMediaWindow::GetDirectory(plugin://plugin.video.horriblesubsviewer/add) failed
09:42:09 T:8156   DEBUG: CGUIMediaWindow::GetDirectory (plugin://plugin.video.horriblesubsviewer/)
09:42:09 T:8156   DEBUG:   ParentPath = [addons://sources/video]
09:42:09 T:8156   DEBUG: Loading items: 2, directory: plugin://plugin.video.horriblesubsviewer/ sort method: 0, ascending: false
09:42:09 T:10404   DEBUG: Thread BackgroundLoader start, auto delete: false
09:42:09 T:10404   DEBUG: Thread BackgroundLoader 10404 terminating
09:42:09 T:4492    INFO: Python script stopped

**Comment by maxdd** [comment]
btw how do you achieve it with rss feeds? i can't find a valid rss reader to torrent addons

**Comment by mancuniancol** [comment]
Please, next time you can share your log with pastebin.com or something else. Sharing in post, it is not good practice. It makes hard to follow the conversation.

Btw, I've fixed for now the problem with HorribleSubs.  They have antibots protection.  Then, the viewer is blocked.  Now, it is working, but I don't know how long will be working :(

The Rss viewer works with:
http://haruhichan.com/feed/feed.php?mode=rss

**Comment by mancuniancol** [comment]
I will wait for your confirmation to close this issue

**Comment by maxdd** [comment]
I still have some problems, removing and adding Haikyuu S2 for example

http://pastebin.com/GffZG1yX

**Comment by mancuniancol** [comment]
Yes, try to turn off the metahandler in the settings of the plugin.

**Comment by mancuniancol** [comment]
Or metainformation

**Comment by mancuniancol** [comment]
I am using an external library that is not longer supported. I need to find a replacement

**Comment by maxdd** [comment]
Now it seems to work ok but i can't delete this anime btw

http://pastebin.com/GefZY9tv

**Comment by mancuniancol** [comment]
OK, I will add to my liat
